spec/bolt/transport/shared_examples.rb

```diff
@@ -327,6 +330,67 @@ def posix_context
     end
   end
 
+  context 'when used by the remote transport' do
+    let(:remote_target) do
+      # TODO: remove the config here. it is a workaround for ????
```

Comment:
I don't know what this comment is saying.

Reply:
BOLT-943 means that we can't add a target with options to the inventory so we have to set config here.